test/Swan.Test/CircularBufferTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Swan.Test
```

Comment:
Remove this file, the class was removed.

Reply:
removed